src/rawtoaces_util/image_converter.cpp

```diff
@@ -576,6 +629,7 @@ bool check_param(
         }
         else
         {
+            on_success(); // TODO Verify with Anton.
```

Comment:
TODO Verify that lack of on_success in this return true case was a bug or intentional.